cmd/Makefile.am

```diff
@@ -465,6 +465,14 @@ libexec_PROGRAMS += snapd-generator/snapd-generator
 snapd_generator_snapd_generator_SOURCES = snapd-generator/main.c
 snapd_generator_snapd_generator_LDADD = libsnap-confine-private.a
 
+##
+## snapd-env-generator
+##
+
+libexec_PROGRAMS += snapd-env-generator/snapd-env-generator
```

Comment:
We could have shipped a shell script :smile:

Reply:
Yes, the first iteration of this PR did that. Its fine if we decide to do. However this program is run for each unit that systemd starts and given how simple it is what we need to do I think C is smaller/faster (and it is saying something if I select C!).

Reply:
Probably as much as ought to be done in C anyway :smile: